### ISSUE TITLE
Why I got this error when I train the model???help me dalao

### ISSUE THREAD

**Issue by Bigwode** [marvis/pytorch-yolo2#87]
.............
9856: nGT 39, recall 0, proposals 0, loss: x 1692.913574, y 1690.920166, w nan, h nan, conf nan, cls nan, total nan
9872: nGT 42, recall 0, proposals 0, loss: x 1690.989502, y 1692.306641, w nan, h nan, conf nan, cls nan, total nan
9888: nGT 50, recall 0, proposals 0, loss: x 1691.757202, y 1691.307861, w nan, h nan, conf nan, cls nan, total nan
9904: nGT 34, recall 0, proposals 0, loss: x 1692.250732, y 1691.642090, w nan, h nan, conf nan, cls nan, total nan
.............

**Comment by Bigwode** [comment]
make the learning rate down to 0.0001 can make it work

**Comment by watrtdc** [comment]
This solution is work well

**Comment by yizhixiaoma** [comment]
@Bigwode  dalao help me!!!  : )  Could you send me your training code ? (295723112@qq.com)
